src/internal.rs

```diff
@@ -421,28 +419,66 @@ impl Rav1dTask {
             ..Default::default()
         }
     }
+}
 
-    pub fn next(&self) -> Rav1dTaskIndex {
-        self.next.load(Ordering::SeqCst)
-    }
-
-    pub fn set_next(&self, next: Rav1dTaskIndex) {
-        self.next.store(next, Ordering::SeqCst)
+impl PartialOrd for Rav1dTask {
+    fn partial_cmp(&self, other: &Self) -> Option<std::cmp::Ordering> {
```

Comment:
Can you import `std::cmp`?  Or if `std::sync::atomic::Ordering` is no longer imported, just import `std::cmp::Ordering`?

Reply:
I'll have a look!

Reply:
We still use `std::sync::atomic::Ordering`; I'm going to import `std::sync::atomic` and `std::cmp` so that we always qualify `Ordering` but never need to fully qualify it with `std::`.

Reply:
Folded into the new https://github.com/memorysafety/rav1d/pull/1447/commits/3a02af0d171821917612e20705d80ee8bbad6623.